Macchiato/palette

```diff
@@ -0,0 +1 @@
+['#494d64', '#ed8796', '#a6da95', '#eed49f', '#8aadf4', '#c6a0f6', '#f5bde6', '#cad3f5', '#6e738d', '#ed8796', '#a6da95', '#eed49f', '#8aadf4', '#c6a0f6', '#f5bde6', '#cad3f5']
```

Comment:
according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#474B61', '#EC8796', '#A7DC96', '#F5DAA3', '#8DB2FB', '#F8BFE9', '#8EDBD0', '#B8C0E0', '#5F647D', '#F98E9E', '#A5D994', '#C7B48A', '#8BAFF7', '#F8BFE9', '#80C4BB', '#A6AFCD']